open-api/rest-catalog-open-api.yaml

```diff
@@ -1343,6 +1344,17 @@ components:
                 type: integer
                 format: int64
 
+    RemoveSnapshotRefUpdate:
+      allOf:
+        - $ref: '#/components/schemas/BaseUpdate'
+        - type: object
+          required:
+            - ref-name
+          properties:
+            name:
+              type: string
+              description: SnapshotRef name to remove
```

Comment:
Nit: Maybe `Name of the snapshot ref to remove`? We tend to prefer plain English sentences in descriptions.